### PR TITLE
Added a CVE and template for missing security headers reporting

### PR DESCRIPTION
**Pull request by socketz**
### Added a CVE and template for missing security headers reporting

- Added HTTP Security Missing Headers for all headers
- Added CVE-2020-11420 (found by myself)
- References:
https://library.e.abb.com/public/ee46f3ff5823400f991ebd9bd43a297e/2CMT2020-005913%20Security%20Advisory%20CS141.pdf

### Template Validation

I've validated this template locally?
- [x] YES
- [ ] NO

### CODE REVIEW

File: miscellaneous/http-present-security-headers.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  description: It searches present security headers, but obviously, could be so less generic and could be useless for Bug Bounty.
+  tags: config,headers
+
+requests:
+  - method: GET
+    path:
+      - "{{BaseURL}}"
+    redirects: true
+    max-redirects: 3
+    extractors:
```
Comment (ehsandeep) [inline miscellaneous/http-present-security-headers.yaml:16]: Hi @socketz, Thanks for adding this and other descriptive templates, do you think we could utilize `matchers:` as opposed to `extractors:`? Do you see any further advantage to printing these that I've missed, given that it adds a lot of information to CLI?

**Comment by princechaddha** [comment]
Also as mentioned in the reference [here](https://library.e.abb.com/public/ee46f3ff5823400f991ebd9bd43a297e/2CMT2020-005913%20Security%20Advisory%20CS141.pdf), we would need valid credentials in order to exploit this vulnerability.

> An attacker with Admin or Engineer login credentials could exploit the vulnerability by manipulating variables that
reference files and by doing this achieve access to files and directories outside the web root folder.

**Comment by socketz** [comment]
I will add the authorization workflow to template. But I have been added a default credentials template related.

File: miscellaneous/http-present-security-headers.yaml (hunk 1 above)
Comment (socketz) [reply miscellaneous/http-present-security-headers.yaml:16]: Hi @ehsandeep, thanks for your comment. That was a test, and I have included it by mistake, I will delete it in another later commit. It is useless 😄 .

**Review by ehsandeep — APPROVED** [review]

**Comment by ehsandeep** [comment]
thank you @socketz, I've updated few templates and moved CVE to another PR - https://github.com/projectdiscovery/nuclei-templates/pull/2567 as we might need to make some changes in the template or additional feature support in nuclei engine to make use of advanced workflow.

**Comment by jub0bs** [comment]
Why are CORS headers treated by this template as _security headers_? The absence of CORS headers from a response is not an indicator of a security issue, since the purpose of CORS is to _relax_ security.

**Pull request merged**